egs/wsj/s5/utils/reverse_lm.sh

```diff
@@ -38,25 +38,13 @@ mkdir -p $outdir
 for f in phones.txt words.txt L.fst L_disambig.fst phones/; do
   cp -r $langdir/$f $outdir
```

Comment:
ok, all that sounds reasonable- try to test it on some ARPA file, and change all except the BOS/EOS filters.